content/md/articles/language/core_overview.md

```diff
@@ -172,15 +193,15 @@ over `def`.
 ```
 
 `if` takes two or three expressions -- a condition expression followed by one
-or two result expressions.
+or two result expressions. If the third expression is not supplied, it defaults to `nil`.
```

Comment:
Perhaps less confusing to say "If the second result expression is not supplied..." so that folks don't have to do the "math"?

Reply:
I think I was being a bit too precious with the existing text - I'm just going to label them as `then` and `else` so it lines up with the syntax block and gets away from the counting altogether.

Reply:
In the paragraph about the else part, it still says "first" expression instead of "condition", in case you want to update that too?

Reply:
thank you for catching that - changed :)